lib/core/versions/0.4.0/OperationProcessor.ts

```diff
@@ -20,81 +20,78 @@ export default class OperationProcessor implements IOperationProcessor {
     anchoredOperationModel: AnchoredOperationModel,
     didDocumentReference: { didDocument: object | undefined }
   ): Promise<ApplyResult> {
-    let operationHash = undefined;
 
     try {
       // NOTE: only used for read interally.
       const didDocument = didDocumentReference.didDocument as (DocumentModel | undefined);
-
       const operation = AnchoredOperation.createAnchoredOperation(anchoredOperationModel);
-      operationHash = operation.operationHash;
 
       if (operation.type === OperationType.Create) {
 
         // If we have seen a previous create operation.
         if (didDocumentReference.didDocument) {
-          return { validOperation: false, operationHash };
+          return { validOperation: false };
         }
 
         const originalDidDocument = Document.from(operation.encodedPayload, this.didMethodName, ProtocolParameters.hashAlgorithmInMultihashCode)!;
 
         const signingKey = Document.getPublicKey(originalDidDocument, operation.signingKeyId);
 
         if (!signingKey) {
-          return { validOperation: false, operationHash };
+          return { validOperation: false };
```

Comment:
nit: some amount of code repetition here, may more readable with a const ApplyResult for valid and invalue, as opposed to looking at the boolean value.

Reply:
Fair point, but will won't fix this because:

1. This is old versioned code, so don't intend to do make any changes unless really needed.
1. We are planning to reset the genesis block in the testnet in the next ION release, at which point this code will be removed forever.